site/content/usage/20-schema.md

```diff
@@ -347,6 +347,10 @@ NodeGroup:
     iam:
       $ref: '#/definitions/NodeGroupIAM'
       $schema: http://json-schema.org/draft-04/schema#
+    instanceName:
+      type: string
+    instancePrefix:
+      type: string
```

Comment:
Do we have to any schema versions bumps after adding new properties?

Reply:
Now that I think about, it may be necessary. However, I don't know how to do it 😄

Reply:
Looking at similar changes to this schema it doesn't look like a version bump is required, so, all good.